File: datasets/binary_tree/type_2_renamed2.py
class Node:
    def __init__(self, x):
        self.x = x
        self.left = None
        self.right = None


def inorder(node):
    if node is None:
        return []

    values = []
    values += inorder(node.left)
    values.append(node.x)
    values += inorder(node.right)

    return values

print("File executed")

File: datasets/java_source-code/binary_tree/type_2_renamed2.java
import java.util.ArrayList;

class Node {
    int x;
    Node left, right;

    public Node(int x) {
        this.x = x;
        left = right = null;
    }
}

public class Main {

    public static ArrayList<Integer> inorder(Node node) {
        ArrayList<Integer> values = new ArrayList<>();

        if (node == null) {
            return values;
        }

        values.addAll(inorder(node.left));
        values.add(node.x);
        values.addAll(inorder(node.right));

        return values;
    }

    public static void main(String[] args) {
        System.out.println("File executed");
    }
}